src/Api/include/Api/LockFreeApiEventProducer.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_API_LOCK_FREE_API_EVENT_PRODUCER_H_
```

Comment:
Our include guards reflect the module/directory name, not the namespace, so drop the `ORBIT_` prefix.

Reply:
Done, thanks.